package.json

```diff
@@ -12,13 +12,13 @@
 		"build": "npm run lint && node ./build.js",
 		"start": "concurrently -k -p \"[{name}]\" -n \"TypeScript,Node\" \"tsc -p ./demo/tsconfig.json -w\" \"ts-node ./demo/server.ts --dist\"",
 		"test": "ts-mocha --require test/testhook.js \"test/specs/**/*.ts\"",
-		"ci-test": "nyc --no-clean ts-mocha --paths \"test/specs/**/*.ts\" -p test/tsconfig.json",
+		"ci-test": "cross-env TS_NODE_PROJECT=test/tsconfig.json nyc --no-clean --require ts-node/register --require tsconfig-paths/register mocha \"test/specs/**/*.ts\" -p test/tsconfig.json",
```

Comment:
Why have we switched from `ts-mocha` to `mocha` for this CI test but not the other CI tests?

Reply:
That's a mistake. We should always do this